lib/crypto/test/crypto_SUITE.erl

```diff
@@ -1027,7 +1027,7 @@ group_config(rsa = Type, Config) ->
                   rsa_oaep(),
                   no_padding()
                  ],
-    Generate = [{rsa, 1024, 3}, {rsa, 2048, 17}, {rsa, 3072, 65537}],
+    Generate = [{rsa, 2048, 3}, {rsa, 3072, 65537}],
```

Comment:
Unfortunatly it still fails on the FIPS-enabled machine.  Maybe 2k should be removed also?